backend/capellacollab/projects/toolmodels/backups/runs/routes.py

```diff
@@ -184,3 +185,6 @@ def get_logs(
         )
 
     return helper.filter_logs(logs, masked_values + masked_values_generated)
+
+
+fastapi_pagination.add_pagination(router)
```

Comment:
Is it required to have this here at the bottom? Otherwise, I would move it to the router definition at the top.

Reply:
I tried it out, but unfortunately, it doesn't work. The error is: 

```py
fastapi_pagination/api.py", line 64, in resolve_params
    raise RuntimeError("Use params or add_pagination") from None
RuntimeError: Use params or add_pagination
```

So it doesn't recognise the pagination registration anymore.